packages/xeus/src/worker.ts

```diff
@@ -116,14 +106,15 @@ export class XeusRemoteKernel {
     }
 
     if (msg_type === 'input_reply') {
-      resolveInputReply(event.msg);
+      // Should never be called as input_reply messages are returned via service worker
```

Comment:
We could explicitly throw an exception here in case this function is ever called in the future.